src/test/java/core/basesyntax/HelloWorldTest.java

```diff
@@ -1,8 +1,115 @@
 package core.basesyntax;
 
-/**
- * Feel free to remove this class and create your own.
- */
+import org.junit.Test;
+import org.junit.BeforeClass;
+import org.junit.Assert;
+
+
 public class HelloWorldTest {
 
-}
\ No newline at end of file
+    private final static double DELTA = 0.001;
+    private static Calculator calculator;
+
+    @BeforeClass
+    public static void setUp() {
+        calculator = new Calculator();
+    }
+
+    @Test
+    public void getSumOfTwoPositiveNumbers() throws BadOperandException {
```

Comment:
```suggestion
    public void getSumOfTwoPositiveNumbers() {
```
fix this in all methods

Reply:
done